### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by DavidVorick** [NebulousLabs/Nodejs-Sia#5]
this package needs documentation about what it is, why it exists, how it is organized, and what to be aware of when modifying it.

**Comment by DavidVorick** [comment]
I feel like the documentation is still incomplete for this package

**Comment by rsabir** [comment]
I agree, for instance I can't find any information about how I can upload or download a content from sia cloud to serve it  to the public (in a nodejs website).

**Comment by songproducer** [comment]
There is some documentation here:
https://blog.sia.tech/api-quickstart-guide-f1d160c05235
